src/web/app/pages-admin/admin-home-page/instructor-data.ts

```diff
@@ -7,6 +9,18 @@ export interface InstructorData {
   institution: string;
   status: string;
   isCurrentlyBeingEdited: boolean;
+  statusCode?: number;
   joinLink?: string;
   message?: string;
 }
+
+/**
+ * Represents the account data associated with a registered instructor.
+ * See registered instructor modal of {@link AdminHomePageComponent}.
+ */
+ export interface RegisteredInstructorAccountData {
+  googleId: string,
+  studentCourses: Course[],
+  instructorCourses: Course[],
+  manageAccountLink: string,
```

Comment:
Just noticed this. Two problems here:
- One space indentation?
- Ends with comma instead of semicolon

Looks like we need an additional ESLint rule here, e.g. https://stackoverflow.com/questions/65388160/how-to-enforce-semicolon-in-typescript-interface

Reply:
Apologies, will fix (by tomorrow) and check the other indentation too. 

Would you like me to add the ESLint rule in this PR? Or should we open another issue for this.

Reply:
Also seems like the issue is that  `export interface RegisteredInstructorAccountData {` is indented to column 1 instead of 0...

Reply:
> Would you like me to add the ESLint rule in this PR? Or should we open another issue for this.

If this is the only violation in the entire repo, ok to add in this PR.

Reply:
Did some preliminary investigation, seems like no matter what config I choose multiple files are affected. I will open another PR for this when I have more time tonight.

One clarification, for this interface:
```diff
export interface SortableTableCellData {
  value?: any; // Optional value used for sorting with sortBy provided in ColumnData
  displayValue?: string; // Raw string to be display in the cell
  style?: string; // Optional value used to set style of data
  customComponent?: {
-   component: Type<any>;
-   componentData: Record<string, any>; // @Input values for component
+   component: Type<any>,
+   componentData: Record<string, any>, // @Input values for component 
  };
}
```
Objects within interfaces should be delimited by comma instead of semicolon right?

Reply:
I've fix indentation issues for those that I could find (as well as standardized the spacings as much as possible).

Reply:
Never really thought about this, I guess we're used to just leaving is to TSLint. Is there a guideline from airbnb on this?

Reply:
I didn't find a definitive resource on this, but currently ESLint classifies the the `customComponent` object as a type literal and not an interface when deciding which rules to apply. I think it would be challenging if we need it to follow the interface and use semicolons instead of commas.

Reply:
Got it. No big deal if we use comma here.